### ISSUE TITLE
Stability with/without listen_addr + public_addr

### ISSUE THREAD

**Issue by voidovic** [input-output-hk/jormungandr#1573]
When is in node config defined listen and public addr, then will node stuck in 5-60 minutes.  

If these directives are commented, then are nodes (passive/active, boths) active without stucking.

**Comment by laplasz** [comment]
which version are you using? v0.8.6 should be better now

**Comment by ofakem-banock** [comment]
I have the same problem. Running v0.8.6, one of my node node is up 22hours now without getting stuck when my node config doesn't have listen and public addr defined. When I put them in my node config, my max uptime is 15min then I get stuck

**Comment by dmindfuse** [comment]
> I have the same problem. Running v0.8.6, one of my node node is up 22hours now without getting stuck when my node config doesn't have listen and public addr defined. When I put them in my node config, my max uptime is 15min then I get stuck

Interesting, I also have the addresses configured. Are you stating that it's better to run without them defined in the config? I haven't tried this, as I thought it was considered bad practice...?

**Comment by ofakem-banock** [comment]
@dmindfuse Yes I am now synced for about 2 days now without the addresses configured. And the blockheight is increasing and it is in sync with pooltool.io and the shelleyexplorer. 

I know it is bad practice but it is the only solution I have at the moment to not get stuck. I run a passive node with the addresses configured and it can't stay synced for more than 1hr. I hope they will look into it and solve this issue

**Comment by dmindfuse** [comment]
@ofakem-banock Glad to hear that’s working for you. I may try that, because the 1hr is spot on the issue I’m seeing as well. Cheers.
